core-rust/state-manager/src/protocol/protocol_configs/mainnet_protocol_config.rs

```diff
@@ -4,7 +4,7 @@ use crate::protocol::*;
 use ProtocolUpdateEnactmentCondition::*;
 
 pub fn mainnet_protocol_config() -> ProtocolConfig {
-    // See config_printer.rs
+    // See config_printer.rs > print_calculated_protocol_config_code()
     ProtocolConfig::new_with_triggers(hashmap! {
```

Comment:
We're missing cuttlefish config for mainnet?

Reply:
Yes - we will need to add it before we publish the node when we know when we want it to actually be :)